src/table/use-table-focus-navigation.ts

```diff
@@ -34,14 +34,14 @@ function useTableFocusNavigation<T extends { editConfig?: TableProps.EditConfig<
 
   const focusableColumns = useMemo(() => {
     const cols = columnDefinitions.map(column => !!column.editConfig);
-    if (selectionType !== 'none') {
+    if (selectionType) {
```

Comment:
What if someone sets it explicitly to "none", `<Table selectionType="none">`?

Reply:
oh, so this is fictional value only in the scope of this utility. ok, great simplification :)